supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -529,7 +529,7 @@
 [discrete]
 [[sudo]]
 ==== image:images/caution.png[with caution] sudo (noun)
-*Description*: `sudo` is a command that allows a user to run a program as another user (the root user by default). When a user requires elevated privileges, using the phrase 'as the root user' prior to a command is preferred over prefixing commands with `sudo`.
+*Description*: `sudo` is a command that allows a user to run a program as another user (the root user by default). When a user requires elevated privileges, use the phrase 'as the root user' before a command instead of prefixing commands with `sudo`.
```

Comment:
CC @bburt-rh since you had volunteered to put together a PR for guidelines on commands that need elevated privileges.

Looks like we have some language here about using "as the root user" instead of prefixing commands with `sudo`. This is kind of the opposite guidance of what we were leaning toward in the meeting. We were leaning towards recommending to use `sudo` in front of the command, when possible, and if not, then make sure to say "as a root user" (not "the").

Not sure if we need to revisit based on this (I wasn't aware of this entry til now), or if we should move forward with as we discussed, and adjust this entry accordingly. Let me know what you think!

Reply:
@bergerhoffer I didn't realize that this info was already here either. Here are couple of things I think we should consider: 
- Because there's only a single root user, "the root user" makes more sense to me than "a root user." 
- I think it's consistent with our discussion last week to recommend to "use the phrase 'as the root user' before a command" over using `sudo`. I like the phrasing here that states that it's a preferred practice rather than a required one because that leaves open the possibility of using `sudo` in some instances.
- That being said, I do think we might want to consider adding an extra sentence along the lines of  "Although you can also use a shell prompt as part of the command to indicate the level of privileges required to run the command  (`#` for root or `$` for regular user), never rely only on the shell prompt to inform users of the required privileges. Above all, use a consistent approach when giving instructions about issuing commands that require elevated privileges."

Reply:
@bburt-rh I thought that the majority of people thought we ought to recommend using sudo when possible as the preferred option. And only if sudo didn't work, then to do the combined prompt + lead-in mention of being the root user.

And I think you're right about "the root user". I think I was misremembering. I believe the discussion was to not say "as root" but instead to say "as the root user".

@redhat-documentation/ccs-style-council please take a look at this, especially those that were in October's meeting where this issue was discussed. Please help us remember what the decision was, and what you think given this information that is already here in the sudo entry.

Reply:
@bergerhoffer  You're right. My own notes say that the guidance should be to use either `sudo` or an introductory phrase and not just to rely on a shell prompt. The consensus was that we wanted something more explicit than a hash mark or dollar sign. Maybe we should change the wording to reflect that we need something more than the shell prompt and that `sudo` or an introductory phrase are acceptable, depending on circumstance and what you need to do to stay consistent with the rest of the doc you're working on.